Index: configure
===================================================================
--- configure	(revision 55858)
+++ configure	(working copy)
@@ -83,6 +83,7 @@
 add_engine agos2 "AGOS 2 games" yes
 add_engine cine "Cinematique evo 1" yes
 add_engine cruise "Cinematique evo 2" yes
+add_engine deskadv "Desktop Adventures" no
 add_engine draci "Dragon History" yes
 add_engine drascula "Drascula: The Vampire Strikes Back" yes
 add_engine gob "Gobli*ns" yes
Index: engines/engines.mk
===================================================================
--- engines/engines.mk	(revision 55858)
+++ engines/engines.mk	(working copy)
@@ -36,6 +36,11 @@
 MODULES += engines/cruise
 endif
 
+ifdef ENABLE_DESKADV
+DEFINES += -DENABLE_DESKADV=$(ENABLE_DESKADV)
+MODULES += engines/deskadv
+endif
+
 ifdef ENABLE_DRACI
 DEFINES += -DENABLE_DRACI=$(ENABLE_DRACI)
 MODULES += engines/draci
Index: base/plugins.cpp
===================================================================
--- base/plugins.cpp	(revision 55858)
+++ base/plugins.cpp	(working copy)
@@ -103,6 +103,9 @@
 		#if PLUGIN_ENABLED_STATIC(CRUISE)
 		LINK_PLUGIN(CRUISE)
 		#endif
+		#if PLUGIN_ENABLED_STATIC(DESKADV)
+		LINK_PLUGIN(DESKADV)
+		#endif
 		#if PLUGIN_ENABLED_STATIC(DRACI)
 		LINK_PLUGIN(DRACI)
 		#endif
